jobs/haproxy/templates/bpm.yml

```diff
@@ -8,11 +8,21 @@ processes:
         writable: true
       - path: /var/vcap/sys/run/haproxy
         writable: true
-<%- if p("ha_proxy.syslog_server") && p("ha_proxy.syslog_server").chars.first == "/" -%>
+
+<%- 
+   additional_volumes = []
+   if p("ha_proxy.syslog_server") && p("ha_proxy.syslog_server").chars.first == "/" 
+      additional_volumes.push({"path" =>p("ha_proxy.syslog_server")})
+   end 
+   if p("ha_proxy.additional_unrestricted_volumes")  
```

Comment:
because `ha_proxy.additional_unrestricted_volumes` has a default of nil, `if p("ha_proxy.additional_unrestricted_volumes")` fails because the property can't be found, so instead you would want `if_p("ha_proxy.additional_unrestricted_volumes") do`

since ha_proxy.syslog_server has a non-null default, it already exists and doesn't run into the same problem using a traditional if statement